pkgs/applications/accessibility/mousetweaks/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchurl, pkg-config
 , glib, gtk3, gnome, gsettings-desktop-schemas, wrapGAppsHook
-, libX11, libXtst, libXfixes, libXcursor
+, xorg
```

Comment:
I am genuinely confusing.

Sometimes I can't use `darwin` and instead I list all the Apple libraries.

But here xorg should be injected as a dependency and its subpackages be specified ad hoc?

Reply:
When can you not use the darwin set